src/+otp/+campbellmooredae/+presets/Canonical.m

```diff
@@ -0,0 +1,15 @@
+classdef Canonical < otp.campbellmooredae.CampbellMooreDAEProblem
+    methods
+        function obj = Canonical()
+            tspan = [0.0, 10*pi];
+            
+            
+            params.r = 10;
+            params.rho = 5;
+            
+            y0 = [15, 0, 0, 0, 0.039524144269346, -0.417592117550000, 0.042213548262602, -0.034980779193327, 0, 0]';
```

Comment:
instead of ' use ;

Reply:
On the initial conditions: aren't they (or rather parts of them), given by equation 37 in the paper?